.github/workflows/verify_windows.yml

```diff
@@ -6,7 +6,7 @@ on:
   pull_request:
     paths:
       - '.github/workflows/verify_windows.yml'
```

Comment:
```suggestion
```
You can just remove this and then you wont have to keep changing the file on commit :)